bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaBridgeHandler.java

```diff
@@ -563,11 +563,23 @@ public void sendCommand(String io, String command, String params, String url) {
 
         Boolean result = sendCommandInternal(io, command, params, url);
         if (!result) {
-            sendCommandInternal(io, command, params, url);
+            scheduler.schedule(() -> {
+                repeatSendCommandInternal(io, command, params, url, thingConfig.getRetries());
+            }, thingConfig.getRetryDelay(), TimeUnit.MILLISECONDS);
         }
     }
 
-    private Boolean sendCommandInternal(String io, String command, String params, String url) {
+    private void repeatSendCommandInternal(String io, String command, String params, String url, int retries) {
+        logger.debug("Retrying command, retries left: {}", retries);
+        boolean result = sendCommandInternal(io, command, params, url);
+        if (!result && (retries > 0)) {
+            scheduler.schedule(() -> {
```

Comment:
You need to cancel the returned future in the handler's `dispose()` method. Otherwise commands might get sent when the thing is not initialized.